db/seeds/development/serials.seeds.rb

```diff
@@ -3,7 +3,7 @@
 
   posts = site1.posts.published.order_by_recently.cycle
 
-  5.times do |i|
+  26.times do |i|
```

Comment:
Yes, I want to show the second page of serials.
I'll add comment, or name magic numbers.